Content.Server/Abilities/Psionics/PsionicAbilitiesSystem.cs

```diff
@@ -290,11 +266,38 @@ private void AddPsionicStatSources(PsionicPowerPrototype proto, PsionicComponent
                 psionic.DampeningSources.Add(proto.Name, proto.DampeningModifier);
         }
 
+        /// <summary>
+        ///     Displays a message to alert the player when they have obtained a new psionic power. These generally will not play for Innate powers.
+        ///     Chat messages of this nature should be written in the first-person.
+        ///     Popup feedback should be no more than a sentence, while the full Initialization Feedback can be as much as a paragraph of text.
+        /// </summary>
+        private void SendFeedbackMessage(EntityUid uid, PsionicPowerPrototype proto, bool playFeedback = true)
+        {
+            if (!playFeedback
+                || !_playerManager.TryGetSessionByEntity(uid, out var session)
+                || session is null)
+                return;
+
+            if (proto.InitializationPopup is null)
+                _popups.PopupEntity(Loc.GetString(GenericInitializationMessage), uid, uid, PopupType.MediumCaution);
+            else _popups.PopupEntity(Loc.GetString(proto.InitializationPopup), uid, uid, PopupType.MediumCaution);
+
+            if (proto.InitializationFeedback is null)
+                return;
+
+            var feedbackMessage = $"[font size={proto.InitializationFeedbackFontSize}][color={proto.InitializationFeedbackColor}]{Loc.GetString(proto.InitializationFeedback)}[/color][/font]";
```

Comment:
It's probably better to do
```suggestion
            if (!Loc.TryGetString(proto.InitializationFeedback, out var feedback))
                return;
            var feedbackMessage = $"[font size={proto.InitializationFeedbackFontSize}][color={proto.InitializationFeedbackColor}]{feedback}[/color][/font]";
```
so you don't display invalid feedbacks

Reply:
I genuinely didn't know this was an option. NEAT!